### PR TITLE
OCPBUGS-27468: features: enabled TranslateStreamCloseWebsocketRequests through TechPreviewNoUpgrade

### PR DESCRIPTION
**Pull request by ingvagabund**
SSIA

### CODE REVIEW

File: config/v1/feature_gates.go [hunk 1]
```diff
@@ -441,4 +441,14 @@ var (
...
 		OwningProduct:       ocpSpecific,
 	}
+
+	FeatureGateTranslateStreamCloseWebsocketRequests = FeatureGateName("TranslateStreamCloseWebsocketRequests")
+	translateStreamCloseWebsocketRequests            = FeatureGateDescription{
+		FeatureGateAttributes: FeatureGateAttributes{
+			Name: FeatureGateTranslateStreamCloseWebsocketRequests,
+		},
+		OwningJiraComponent: "kube-apiserver",
+		ResponsiblePerson:   "apiserver-team",
```
[the remark this answers is not in the document]
Comment (ingvagabund) [reply config/v1/feature_gates.go]: Done

**Comment by JoelSpeed** [comment]
@ingvagabund Given this is an upstream feature, I assume it's currently disabled upstream in 1.29? Is there a specific/good E2E suite that we can run to give it a test, or will the general E2E suite be sufficient here?

**Comment by ingvagabund** [comment]
Yes, introduced in 1.28 and disabled by default since. I'd like to backport it into 4.15 as well so customers can test the functinality sooner.

> Is there a specific/good E2E suite that we can run to give it a test, or will the general E2E suite be sufficient here?

No specific tests to run to my knowledge. We will be looking at the testing side in 4.16.

**Comment by JoelSpeed** [comment]
Ack, so long as E2E passes, I'm cool with this then, thanks for the background context
/lgtm

**Pull request merged**